README.md

```diff
@@ -0,0 +1,85 @@
+# The yt Project
+
+[![Users' Mailing List](https://img.shields.io/badge/Users-List-lightgrey.svg)](http://lists.spacepope.org/listinfo.cgi/yt-users-spacepope.org/)
+[![Devel Mailing List](https://img.shields.io/badge/Devel-List-lightgrey.svg)](http://lists.spacepope.org/listinfo.cgi/yt-dev-spacepope.org/)
+[![Build Status](https://img.shields.io/travis/yt-project/yt.svg?branch=master)](https://travis-ci.org/yt-project/yt)
+[![Latest Documentation](https://img.shields.io/badge/docs-latest-brightgreen.svg)](http://yt-project.org/docs/dev/)
+[![Data Hub](https://img.shields.io/badge/data-hub-orange.svg)](https://hub.yt/)
+                
+[![yt logo](doc/source/_static/yt_logo.png "yt logo")](http://yt-project.org/)
+
+yt is an open-source, permissively-licensed python package for analyzing and
+visualizing volumetric data.
+
+yt supports structured, variable-resolution meshes, unstructured meshes, and
+discrete or sampled data such as particles. Focused on driving
+physically-meaningful inquiry, yt has been applied in domains such as
+astrophysics, seismology, nuclear engineering, molecular dynamics, and
+oceanography. Composed of a friendly community of users and developers, we want
+to make it easy to use and develop — we'd love it if you got involved!
+
+We've written a [method
+paper](http://adsabs.harvard.edu/abs/2011ApJS..192....9T) you may be interested
+in; if you use yt in the preparation of a publication, please consider citing
+it.
+
+## Installation
+
+If you're using conda with [conda-forge](http://conda-forge.github.io/), you
+can install the most recent stable version by running:
+
+```
+conda install yt
+```
+
+Or, if you want the latest nightly build, you can manually install from our
+repository:
+
+```
+conda install -c http://use.yt/with_conda yt
+```
+
+To get set up with a development version, you can clone this repository and
+install like this:
+
+```
+git clone https://github.com/yt-project/yt
```

Comment:
This is what I have done personally, but it's not exactly in line with https://github.com/yt-project/yt/pull/1378 (and the conversation @ngoldbaum and I had about it), where we name the local repo `yt-git`.
Wondering if this might cause confusion down the road...